corehq/apps/sso/utils/user_helpers.py

```diff
@@ -0,0 +1,15 @@
+def get_email_domain_from_username(username):
```

Comment:
Do we have a common place to put generally useful utility functions for the webapp? It looks like this residing in SSO already reduces duplication there, but I'd imagine this functionality is duplicated in other parts, and it would be weird for HQWebapp to depend on SSO just to pull this in.

Reply:
+1

My typical approach to things like this is to put them in the specific place where they are initially introduced and used until they are wanted somewhere else, at which point a refactor is done to move it to a more commonly accessible place. However this one does seem generally useful enough that it probably makes sense to put it in the commonly accessible place right away. The move could easily be done in a separate PR though.